### ISSUE TITLE
Cannot Select Other Characters or NPC's in Gpose

### ISSUE THREAD

**Issue by Tatsuo16** [imchillin/Anamnesis#693]
Description:
While in gpose, I cannot select, interact, or edit any pinned players, npc's, etc. with Anamnesis. If I try to select a pinned actor while in gpose, I will receive a, "This actor does not have a model," error message. This not only includes being in a crowded area like Limsa, but also in my apartment with only myself and a mannequin as selectable actors. This is the first time I have tried to use Anamnesis since Endwalker came out. The only changes to my methodology are the updates themselves to both FFXIV and Anamnesis. Anamnesis is fully updated prior to this error.

Steps to Reproduce:
1. Pin any Player, npc, etc. in Anamnesis before entering gpose.
2. Enter gpose
3. Try to select the pinned actor but have Anamnesis refuse to switch to the pinned actor. You also will receive the, "This actor does not have a model," error message. Screenshots will be included only on the exception that they may add context to the errors reported. 

Expected Behavior:
The expected behavior was to be able to switch to any pinned actor and edit their poses while in gpose. 

Log Files:
[2021-12-15-15-04-03.txt](https://github.com/imchillin/Anamnesis/files/7722611/2021-12-15-15-04-03.txt)
[2021-12-15-15-34-50.txt](https://github.com/imchillin/Anamnesis/files/7722612/2021-12-15-15-34-50.txt)
[2021-12-15-15-34-59.txt](https://github.com/imchillin/Anamnesis/files/7722613/2021-12-15-15-34-59.txt)
[2021-12-15-15-35-21.txt](https://github.com/imchillin/Anamnesis/files/7722614/2021-12-15-15-35-21.txt)
[2021-12-15-15-36-14.txt](https://github.com/imchillin/Anamnesis/files/7722615/2021-12-15-15-36-14.txt)
[2021-12-15-13-17-04.txt](https://github.com/imchillin/Anamnesis/files/7722616/2021-12-15-13-17-04.txt)
[2021-12-15-13-17-10.txt](https://github.com/imchillin/Anamnesis/files/7722617/2021-12-15-13-17-10.txt)
[2021-12-15-14-20-08.txt](https://github.com/imchillin/Anamnesis/files/7722618/2021-12-15-14-20-08.txt)
[2021-12-15-14-43-48.txt](https://github.com/imchillin/Anamnesis/files/7722619/2021-12-15-14-43-48.txt)
![Screenshot (56)](https://user-images.githubusercontent.com/74097700/146263736-4643e76f-6a30-48a2-976e-bc2efcb11689.png)
![Screenshot (57)](https://user-images.githubusercontent.com/74097700/146263746-09ad3495-498c-47ad-a233-44f701183312.png)
![Screenshot (58)](https://user-images.githubusercontent.com/74097700/146263747-17da92d3-0cd3-419e-b90c-bc72cd37e4c1.png)

**Comment by Yuki-Walsh** [comment]
when this occurs, can you confirm that you can target them in-game?

**Comment by Tatsuo16** [comment]
Well, before all the updates dropped, in gpose I could with both cmtools and Anamnesis. I was editing poses for other players, npc’s, etc just fine. Now I can’t seem to do so. I’m not sure if I’m doing something wrong on my end, or if it’s the software itself. I just know besides the updates, nothing else changed. 

The only things I might do for now is redownload the two programs after I sleep and see if that fixes it. I also imported the mods I used before. I have no idea if that would even have an impact with the posing software. 

Hope this helps. Let me know if you have any other questions or if I misunderstood something. 

From Joshua Sparrow
